app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueModal.jsx

```diff
@@ -337,23 +326,16 @@ export class LinkIssueModal extends Component {
       : formatMessage(messages.linkIssue);
   };
 
-  setModalState = (newModalState) => {
-    this.setState({
-      modalState: {
-        ...this.state.modalState,
-        ...newModalState,
-      },
-    });
+  setItems = (newState) => {
+    this.setState({ newState });
```

Comment:
May be you mean `this.setState(newState)`?
Also please check it for Unlink/Post issue modals.